blender/extension/ui/import_pdb_operator.py

```diff
@@ -109,7 +112,21 @@ def draw_alpha_carbons(self, universe, collection_protein, domain_regions):
             collection_protein.children.link(collection_domain)
             collection_domain.objects.link(atom_mesh_object)
 
-            # UV balls
+            if add_convex_hull:
+                # Create the convex hull
+                bm = bmesh.new()
+                bm.from_mesh(atom_mesh_object.data)
+                hull = bmesh.ops.convex_hull(bm, input=bm.verts)
+                bm.to_mesh(atom_mesh_object.data)
+                bm.free()
+
+                # We can choose to color the hull:
+                atom_mesh_object.active_material = material
+
+                # We can make the outer points invisible:
+                radius = 0
```

Comment:
These last two code lines are optional -- coloring the hull is not great and we might remove it. If we do remove it, showing the points could color-code them. Hard to say what's best UX. This could also be extra options, but it's minor enough that it doesn't matter.